README.rst

```diff
@@ -7,8 +7,8 @@ Knack
 .. image:: https://img.shields.io/pypi/pyversions/knack.svg
     :target: https://pypi.python.org/pypi/knack
 
-.. image:: https://travis-ci.org/Microsoft/knack.svg?branch=master
-    :target: https://travis-ci.org/Microsoft/knack
+.. image:: https://dev.azure.com/azure-sdk/public/_apis/build/status/cli/microsoft.knack?branchName=master
+    :target: https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=1643&branchName=master
```

Comment:
`branchName` is still `master` instead of changing to `dev`?

Reply:
Kindly refresh it.